client/src/shared/components/Text-field/text-field.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+interface Props {
+  label?: string;
+  placeholder?: string;
```

Comment:
- please add default props for `label` and `placeholder`. empty string is okay
- please run prettier in the file. I think it's not well formatted.